sycl/include/sycl/ext/oneapi/reduction.hpp

```diff
@@ -2242,9 +2158,11 @@ reduCGFuncImplAtomic64(handler &CGH, KernelType KernelFunc,
 // IsReduOptForAtomic64Add, as prescribed in its documentation, should then also
 // be made.
 template <typename KernelName, typename KernelType, int Dims, class Reduction>
-enable_if_t<Reduction::has_atomic_add_float64>
-reduCGFuncAtomic64(handler &CGH, KernelType KernelFunc,
-                   const nd_range<Dims> &Range, Reduction &Redu) {
+void reduCGFuncAtomic64(handler &CGH, KernelType KernelFunc,
+                        const nd_range<Dims> &Range, Reduction &Redu) {
+  // static_assert(
+  //     Reduction::has_atomic_add_float64,
+  //     "Expected to be called for reductions with atomic add FP64 support!");
```

Comment:
Is this needed?

Reply:
There is same exact assert inside `reduCGFuncImplAtomic64`. Removing it from here.